packages/react-app/src/services/uniswap.service.ts

```diff
@@ -50,50 +53,75 @@ export async function fetchPairWithStables(tokenA: string): Promise<PairModel[]
   });
   return mapPairList(queryResult.pairs);
 }
+// #endregion
+
+// #region Contract Queries
 
 export async function fetchRoutePairWithStable(tokenA: string) {
   const { chainId, isTestNetwork } = getNetwork();
+  const PRICE_ZERO = Promise.resolve({ price: isTestNetwork ? '1' : '0' }); // Fallback to 1 for dev or 0 for production
```

Comment:
Is that resolving this card : #198 Handle TOken Price returning zero ?

Reply:
Yah i think yes!